en/docs/services/service-concepts.md

```diff
@@ -1,46 +1,43 @@
 # Service Concepts
 
-Before you start creating services with Choreo, there are basic concepts that you need to understand. This section walks you through all the service concepts you need to know.
+Choreo services allow you to create enterprise integration services incorporating complex business logic in a user-friendly low-code environment. Here are some basic concepts which will help you understand Choreo services as you develop your services in the [Choreo Console]({{choreo_console}}).
  
 ## Service
-A service in Choreo is a logical representation of a repeatable activity that has a specified outcome.
-
-## Service Sample
-A service sample in Choreo is a prebuilt service that performs a specific activity. Choreo provides a comprehensive set of service samples for you to clone, edit, and try out, depending on your requirement. 
+A service in Choreo is an enterprise integration service exposed via a REST API. Choreo provides a comprehensive set of pre-built service samples for you to clone, edit, and try out, depending on your requirement. 
 
 ## API Call
-An API call in Choreo services allows you to connect to and interact with an external service through your service. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
+An API call in Choreo allows you to connect to an external service through your Choreo service. You can use an API call to interact with generic protocol-based APIs and application-specific APIs.
 
 Following are the connections you can use to interact with generic protocol-based APIs:
   
- - **HTTP**: To interact with the HTTP API and communicate with an external endpoint via the HTTP protocol.   
- - **SMTP**: To send emails via the SMTP protocol.   
- - **POP3**: To receive emails from a POP3 email server.  
- - **IMAP**: To receive emails from any email server that supports the IMAP protocol. 
+ - **HTTP**: Communicates with external endpoints over HTTP.   
+ - **SMTP**: Performs email-related operations over SMTP. 
```

Comment:
We have used `-` here, but not in the integration concepts page. Please correct.

Reply:
fixed.